coalib/parsing/DefaultArgParser.py

```diff
@@ -184,7 +184,7 @@ def default_arg_parser(formatter_class=None):
         help='list all bears')
 
     outputs_group.add_argument(
-        '-l', '--filter-by-language', nargs='+', metavar='LANG',
+        '-l', 'filter-by language', nargs='+', metavar='LANG',
```

Comment:
here should be `--filter-by-language` that's the argument that you write on the console. don't change it